build_image.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+oc create -f deploy/crds/neutron_v1_neutronovsagent_crd.yaml
+oc create -f deploy/crds/neutron_v1_neutronsriovagent_crd.yaml
+oc create -f deploy/crds/neutron.openstack.org_ovsnodeosps_crd.yaml
+oc create -f deploy/crds/neutron.openstack.org_ovncontrollers_crd.yaml
+make
+operator-sdk build --image-builder buildah "${REGISTRY:-quay.io/openstack-k8s-operators/neutron-operator:v0.0.3}"
```

Comment:
I might make this variable more specific. REGISTRY isn't quite correct. Its more of a BUILD_IMAGE right?

Also, I do think we might want to be careful people don't run this and accidentally push to openstack-k8s-operators right? It might be better to just require it to be set.

Reply:
Name changed. 
About this default, only teammates can push to quay.io/openstack-k8 so don't see a risk here, also you still need to set REGISTRY_AUTH